docs/guide/getstarted/configuration.md

```diff
@@ -97,6 +97,12 @@ Saves a screenshot to a given path if the Selenium driver crashes
 Type: `String`|`null`<br>
 Default: *null*
 
+### screenshotOnReject
+Attaches a screenshot of a current page to the error if the Selenium driver crashes
+
+Type: `Boolean`<br>
+Default: *false*
+
```

Comment:
yep, this is all about performance. Someone can launch tests without any gui report. I don't think he need to spent extra time to get screenshot and extra memory to store it